runtimes/mlflow/tests/test_encoding.py

```diff
@@ -1,32 +1,38 @@
-import pytest
 import numpy as np
-
-from mlserver.codecs.numpy import _to_datatype
-
+import pandas as pd
+import pytest
 from mlserver_mlflow.encoding import (
     MLflowPayload,
-    DefaultOutputName,
     to_outputs,
+    _convert_to_tensor_data_if_raw,
 )
 
+from mlserver.codecs.numpy import _to_datatype
+
 
 @pytest.mark.parametrize(
     "mlflow_payload",
     [
         np.array([1, 2, 3]),
         {"foo": np.array([1, 2, 3])},
         {"foo": np.array([1, 2, 3]), "bar": np.array([4, 5, 6], dtype=np.float32)},
+        pd.DataFrame([[1, 2, 3]]),
```

Comment:
@adriangonz what edge cases we need to consider here for pandas?

Reply:
Mmmh good question... should we add some for multi-column dataframes maybe?

Reply:
added